src/main/java/org/isf/dicom/manager/SourceFiles.java

```diff
@@ -346,24 +347,24 @@ public static synchronized void loadDicom(FileDicom dicomFileDetail, File source
 				try {
 					studyDate = studyDate != null ? studyDate : dicomObject.getDate(Tag.StudyDate, Tag.StudyTime);
 				} catch (Exception ecc) {
-					System.out.println("DICOM: Unparsable StudyDate");
+					LOGGER.error("DICOM: Unparsable StudyDate");
 				}
 				try {
 					seriesDate = seriesDate != null ? seriesDate : dicomObject.getDate(Tag.SeriesDate, Tag.SeriesTime);
 				} catch (Exception ecc) {
-					System.out.println("DICOM: Unparsable StudyDate");
+					LOGGER.error("DICOM: Unparsable SeriesDate");
 				}
 
 				//set by DICOM properties
 				patientID = dicomObject.getString(Tag.PatientID) == null ? patientID : dicomObject.getString(Tag.PatientID);
 				patientName = dicomObject.getString(Tag.PatientName) == null ? patientName : dicomObject.getString(Tag.PatientName);
 				patientAddress = dicomObject.getString(Tag.PatientAddress) == null ? patientAddress : dicomObject.getString(Tag.PatientAddress);
 				patientAge = dicomObject.getString(Tag.PatientAge) == null ? patientAge : dicomObject.getString(Tag.PatientAge);
-				String acquisitionsInSeries = dicomObject.getString(Tag.AcquisitionsInSeries);
-				String acquisitionsInStudy = dicomObject.getString(Tag.AcquisitionsInStudy);
-				String applicatorDescription = dicomObject.getString(Tag.ApplicatorDescription);
-				String dicomMediaRetrievalSequence = dicomObject.getString(Tag.DICOMMediaRetrievalSequence);
-				String patientComments = dicomObject.getString(Tag.PatientComments);
+				//String acquisitionsInSeries = dicomObject.getString(Tag.AcquisitionsInSeries);
+				//String acquisitionsInStudy = dicomObject.getString(Tag.AcquisitionsInStudy);
+				//String applicatorDescription = dicomObject.getString(Tag.ApplicatorDescription);
+				//String dicomMediaRetrievalSequence = dicomObject.getString(Tag.DICOMMediaRetrievalSequence);
+				//String patientComments = dicomObject.getString(Tag.PatientComments);
```

Comment:
Why these comments?

Reply:
Because they are variables that aren't used in the method; but I left them there in case someone knew why they were included. I am more than willing to remove them.